entrypoint.sh

```diff
@@ -27,6 +27,8 @@ function main {
 
   shift $(( OPTIND - 1 ))
 
+  validate_md_suffixes
```

Comment:
Can we move this to somewhere around Line 172 so that the checks that kill the script are all in the same-ish place?

Reply:
Done.